src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -360,6 +384,8 @@ object PeerActor {
 
   case class SendMessage[M <: Message](message: M)(implicit val enc: RLPEncoder[M])
 
+  case class Broadcast[M <: Message](block: M)
```

Comment:
I was thinking about that, but it would create circular dependency also peer manager only passes Broadcast to other peers